students/billy_galloway/lesson_3/assignment/customer_db/basic_operations.py

```diff
@@ -0,0 +1,90 @@
+'''
+basic operations for employees to perform
+for customer data
+'''
+
+import logging
+from peewee import *
+from customer_model import *
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+logger.info('Working with Customer class')
+
+CUST_ID = 0
+NAME = 1
+LAST_NAME = 2
+HOME_ADDRESS = 3
+EMAIL_ADDRESS = 4
+PHONE = 5
+STATUS = 6
+CREDIT_LIMIT = 7
+
+def add_customer(customer_id, name, last_name,
+                 home_address, email_address,
+                 phone_number, status, credit_limit):
+    ''' add new customers to the database '''
+    customer = (customer_id, name, last_name,
+                home_address, email_address,
+                phone_number, status, credit_limit)
+    try:
+        with database.transaction():
+            new_customer = Customer.create(
+                customer_id=customer[CUST_ID],
+                name=customer[NAME],
+                last_name=customer[LAST_NAME],
+                home_address=customer[HOME_ADDRESS],
+                email_address=customer[EMAIL_ADDRESS],
+                phone_number=customer[PHONE],
+                status=customer[STATUS],
+                credit_limit=customer[CREDIT_LIMIT])
+            new_customer.save()
+            logger.info(f'{name} {last_name} added to database')
+    except Exception as e:
```

Comment:
Avoid using bare exceptions like this. A general exception trap will capture (and handled) anything that raises an exception, which poses a significant risk for released code. No need to make a change for this assignment, but keep it in mind for future submissions.

-Luis